Editor/Resources/Font/FontAwesome SDF.asset

```diff
@@ -0,0 +1,257 @@
+%YAML 1.1
```

Comment:
Is this a font? Where is this being pulled from? We have restrictions on using licensed resources (fonts/images etc) so want to confirm